contracts/contracts/layer_0/clearing/ClearingStorageWrapper2.sol

```diff
@@ -868,46 +936,100 @@ abstract contract ClearingStorageWrapper2 is HoldStorageWrapper2 {
             _getClearedAmountForByPartition(_partition, _tokenHolder) * factor;
     }
 
-    function _getClearingForByPartitionAdjusted(
-        IClearing.ClearingOperationIdentifier
-            calldata _clearingOperationIdentifier
+    function _getClearingTransferForByPartitionAdjusted(
+        bytes32 _partition,
+        address _tokenHolder,
+        uint256 _clearingId
     )
         internal
         view
         returns (
-            uint256 amount_,
-            uint256 expirationTimestamp_,
-            address destination_,
-            IClearing.ClearingOperationType clearingOperationType_,
-            bytes memory data_,
-            bytes memory operatorData_,
-            IHold.Hold memory hold_
+            IClearingTransfer.ClearingTransferData memory clearingTransferData_
         )
     {
         uint256 factor = _calculateFactor(
             _getAbafAdjusted(),
             _getClearingLabafByPartition(
-                _clearingOperationIdentifier.partition,
-                _clearingOperationIdentifier.clearingId,
-                _clearingOperationIdentifier.tokenHolder
+                _buildClearingOperationIdentifier(
+                    _tokenHolder,
+                    _partition,
+                    _clearingId,
+                    IClearing.ClearingOperationType.Transfer
+                )
             )
         );
-        (
-            amount_,
-            expirationTimestamp_,
-            destination_,
-            clearingOperationType_,
-            data_,
-            operatorData_,
-            hold_
-        ) = _getClearingForByPartition(_clearingOperationIdentifier);
-        amount_ *= factor;
+        clearingTransferData_ = _getClearingTransferForByPartition(
+            _partition,
+            _tokenHolder,
+            _clearingId
+        );
+        clearingTransferData_.amount *= factor;
```

Comment:
To me we are defining two variables of 1 usage. Increases binary size and more gas cost.